definitions/contested/json/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -18094,6 +18094,13 @@
     "UserRole": "caseworker-divorce-financialremedy-judiciary",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/02/2021",
+    "CaseTypeID": "FinancialRemedyContested",
+    "CaseFieldID": "caseFileViewComponentLauncher",
+    "UserRole": "caseworker-divorce-financialremedy-solicitor",
+    "CRUD": "CRU"
```

Comment:
Sorry I probably should have spotted this on the non-prod PR. Perhaps the CRU isn't a good idea. I think it would give solicitors the ability to move documents. Would "R" be enough to give them access and no more?

Reply:
That's right! Making this Read only prevents the solicitors from moving docs in CFV. [73c146a](https://github.com/hmcts/finrem-ccd-definitions/pull/1954/commits/73c146a988f5aa095787ef769423f609299aff09)